chain/chain/src/runtime/mod.rs

```diff
@@ -192,21 +191,8 @@ impl NightshadeRuntime {
                    next_block_epoch_start = epoch_manager.is_next_block_epoch_start(prev_block_hash).unwrap()
             );
 
-            let mut slashing_info: HashMap<_, _> = challenges_result
-                .iter()
-                .filter_map(|s| {
-                    if shard_layout.account_id_to_shard_id(&s.account_id) == shard_id
-                        && !s.is_double_sign
-                    {
-                        Some((s.account_id.clone(), None))
-                    } else {
-                        None
-                    }
-                })
-                .collect();
-
             if epoch_manager.is_next_block_epoch_start(prev_block_hash)? {
-                let (stake_info, validator_reward, double_sign_slashing_info) =
+                let (stake_info, validator_reward, _double_sign_slashing_info) =
```

Comment:
Assuming we'll get rid of `_double_sign_slashing_info` in a future PR?

Reply:
Yes, in subsequent PRs.